rustworkx/digraph6.py

```diff
@@ -0,0 +1,30 @@
+"""digraph6 format helpers.
```

Comment:
Right now, this wrapper is pointless:
* For writing, see https://github.com/Qiskit/rustworkx/blob/7318a803dc87853cbb2e916d7095b440985507e0/rustworkx/__init__.py#L2285 for the correct pattern
* For reading, if the file format contains the hint: use the same pattern from https://github.com/Qiskit/rustworkx/blob/7318a803dc87853cbb2e916d7095b440985507e0/src/graphml.rs#L1248. If it doesn't, then just have two functions `digraph_read_graph6` and `graph_read_graph6`